src/dashboard/Data/Browser/BrowserToolbar.react.js

```diff
@@ -71,9 +72,19 @@ let BrowserToolbar = ({
     menu = (
       <BrowserMenu title='Edit' icon='edit-solid'>
         <MenuItem
+          text="Add a row"
```

Comment:
Yeah, I wrote this before realizing that the button was adding a row and attaching it. This should also say "Create a ${targetClass} and attach" for clarity, I think.